src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -117,7 +117,7 @@ pub fn is_const_fn(tcx: TyCtxt, def_id: DefId) -> bool {
             false
         }
     } else {
-        tcx.sess.cstore.is_const_fn(def_id)
+        tcx.is_const_fn(def_id)
```

Comment:
You could replace all of these "check locally or externally if a `fn` is `const`" helpers if you had `is_const_fn` also for the local crate.